README.md

```diff
@@ -1,149 +1,194 @@
 # sgrep
 
-[![r2c community slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=for-the-badge&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
+[![CircleCI](https://circleci.com/gh/returntocorp/sgrep.svg?style=svg)](https://circleci.com/gh/returntocorp/sgrep)
+[![r2c Community Slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=flat&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
 
-[sgrep.live](https://sgrep.live/): try it now!
+`sgrep` is a tool for easily detecting and preventing bugs and anti-patterns in
+your codebase. It combines the convenience of `grep` with the correctness of
+syntactical and semantic search. Quickly write rules so you can code with
+confidence.
 
-sgrep, for syntactical \(and occasionnally semantic\) grep, is a tool to help find bugs by specifying code patterns using a familiar syntax. The idea is to mix the convenience of grep with the correctness and precision of a compiler frontend.
+Try it now: [https://sgrep.live](https://sgrep.live/)
 
-## Quick Examples
+## Overview
 
-| **pattern** | **will match code like** |
+Simple patterns for finding code in many languages:
```

Comment:
suggestion for this table, it should show the sgrep invocation exactly to make it clear how to use the patteern, and have examples in Javascript and golang as well, eg:

command          | matches 
`sgrep -lang js const x = 5`     | const $FOO = $BAR

Reply:
Oh, and ideally we would actually link each row to a sgrep.live example :D

Reply:
Roger that, I'll iterate here.

Reply:
I added a JS and Go example. As for showing the full invocation, adding that much text turns these examples into multi-line rows. Shall we just stick with the 5 simple examples for now and iterate on this table more specifically in the future?

Reply:
@mschwager sgtm!

Reply:
I dig it! Probably should stay away from the multiline wrap if we can, defer to you there. Let's just make sure that it's obvious later on how to use "pattern"